pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>uk.gov.ons.ctp.product</groupId>
   <artifactId>rm-common-config</artifactId>
-  <version>10.49.13-SNAPSHOT</version>
+  <version>10.49.14</version>
```

Comment:
Don't need to bump this. It will automatically bump on merge

Reply:
Noted and won't check changes like this in next time. I'm referencing a specific version in the other projects, so I thought it made sense to bump the version like this